packages/message-switch/message-switch.1.4.0/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "1.2"
+maintainer: "xen-api@lists.xen.org"
+authors: [ "xen-api@lists.xen.org" ]
+homepage: "https://github.com/xapi-project/message-switch"
+bug-reports: "https://github.com/xapi-project/message-switch/issues"
+dev-repo: "git://github.com/xapi-project/message-switch"
+tags: [ "org:xapi-project" ]
+build: [
+  ["./configure" "--bindir" "%{bin}%"]
+  [make]
+]
+install: [
+  [make "install"]
+]
+remove: [
+  ["ocamlfind" "remove" "message_switch"]
+]
+depends: [
+  "oasis" {build}
+  "ocamlfind" {build}
+  "base-unix"
+  "cohttp" {>= "0.15.0" & < "0.22.0"}
+  "rpc" {>= "1.9.51"}
+  "sexplib"
+  "ppx_sexp_conv"
+  "uri"
+  "re"
+  "mtime" {< "1.0.0"}
+  "mirage-block-unix" {>= "2.4.0"}
+  "shared-block-ring" {>= "2.3.0"}
+  "cmdliner"
+  "async"
```

Comment:
this needs `"async" {<"v0.9.0"}` due to

```
#=== ERROR while installing message-switch.1.4.0 ==============================#
# opam-version         1.2.2 (58ef3b8213100953848d362f7120a30356d7f77d)
# os                   linux
# command              make
# path                 /home/opam/.opam/4.03.0/build/message-switch.1.4.0
# compiler             4.03.0
# exit-code            2
# env-file             /home/opam/.opam/4.03.0/build/message-switch.1.4.0/message-switch-29-9eeefe.env
# stdout-file          /home/opam/.opam/4.03.0/build/message-switch.1.4.0/message-switch-29-9eeefe.out
# stderr-file          /home/opam/.opam/4.03.0/build/message-switch.1.4.0/message-switch-29-9eeefe.err
### stdout ###
# [...]
# File "async/protocol_async.mli", line 16, characters 5-13:
# Warning 3: deprecated: module Core.Std
# [since 2017-02] Use Core. The Std sub-module is no longer needed
# File "async/protocol_async.mli", line 17, characters 5-14:
# Warning 3: deprecated: module Async.Std
# [since 2017-02] use Async. The Std sub-module is no longer needed
# File "async/protocol_async.mli", line 1:
# Error: Some fatal warnings were triggered (2 occurrences)
# Command exited with code 2.
# Makefile:34: recipe for target 'build' failed
### stderr ###
# W: Cannot find source file matching module 'message_switch_server' in library message_switch_server.
# W: Use InterfacePatterns or ImplementationPatterns to define this file with feature "source_patterns".
# W: Cannot find source file matching module 'message_switch' in library message_switch.
# W: Use InterfacePatterns or ImplementationPatterns to define this file with feature "source_patterns".
# E: Failure("Command ''/home/opam/.opam/4.03.0/bin/ocamlbuild' core/message_switch.cma core/message_switch.cmxa core/message_switch.a core/message_switch.cmxs lwt/message_switch_lwt.cma lwt/message_switch_lwt.cmxa lwt/message_switch_lwt.a lwt/message_switch_lwt.cmxs async/message_switch_async.cma async/message_switch_async.cmxa async/message_switch_async.a async/message_switch_async.cmxs unix/message_switch_unix.cma unix/message_switch_unix.cmxa unix/message_switch_unix.a unix/message_switch_unix.cmxs switch/message_switch_server.cma switch/message_switch_server.cmxa switch/message_switch_server.a switch/message_switch_server.cmxs cli/main.native core_test/link_test_main.native core_test/client_main.native core_test/client_async_main.native core_test/client_unix_main.native core_test/server_main.native core_test/server_async_main.native core_test/server_unix_main.native switch/switch_main.native -tag debug -tag tests -j 4' terminated with error code 10")
```

we also highly recommend turning off warnings-as-errors for opam (but leaving on for development)